src/plays/index.js

```diff
@@ -0,0 +1,32 @@
+// Do not modify the content of this file
```

Comment:
This file shouldn't be checked in. The current process flow creates this file dynamically. Your implementation might have started before the `create-workflow` merge and that could be the reason it showed up in git. Please discard it.